### PR TITLE
[MODORG-46] Update dependent raml-util

### PR DESCRIPTION
**Pull request by bekhzodkurbonboev**
## Purpose
Resolves: [MODORG-46](https://issues.folio.org/browse/MODORG-46) Update dependent raml-util

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate action.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code are 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day 
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, 
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by Dmitriy-Butramyou — APPROVED** [review]

File: src/main/java/org/folio/rest/impl/OrganizationApi.java [hunk 1]
```diff
@@ -33,14 +33,14 @@ public OrganizationApi() {
   }
 
   @Override
-  public void getOrganizationsOrganizations(String query, int offset, int limit, String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    organizationService.getOrganizationCollection(offset, limit, query, lang, vertxContext, okapiHeaders)
+  public void getOrganizationsOrganizations(String query, String totalRecords, int offset, int limit, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```
Comment (Abdulkhakimov) [inline src/main/java/org/folio/rest/impl/OrganizationApi.java:36]: Is totalRecords something new stuff which bringed after raml-util upgrade?

Comment (Dmitriy-Butramyou) [reply src/main/java/org/folio/rest/impl/OrganizationApi.java:36]: Yes

**Comment by bekhzodkurbonboev** [comment]
I run Karate tests against locally deployed module, 'OrganizationsApiTest' passed successfully 
image
But was not able to run 'CrossModulesApiTest', this one is failing in set-up.

File: src/main/java/org/folio/rest/impl/OrganizationApi.java (hunk 1 above)
Comment (grabsefx) [reply src/main/java/org/folio/rest/impl/OrganizationApi.java:36]: please refer to https://issues.folio.org/browse/RMB-724

**Review by grabsefx — APPROVED** [review]

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**